environments/common/inventory/group_vars/all/openondemand.yml

```diff
@@ -7,6 +7,11 @@
 
 # openondemand_servername: '' # Must be defined when using openondemand
 
+# regex defining hosts which openondemand can proxy; the default regex is compute nodes (for apps) and grafana host.
+# the autogenerated regex may need overriding if compute nodes names do not contain numbers in a consistent position or
+# or include regex special characters.
```

Comment:
Could you include an example of what the regex looks like in its final state here as well as in the docstring on the filter? Might be helpful for debugging. Not a showstopper though...